### ISSUE TITLE
Issue with color picker

### ISSUE THREAD

**Issue by marionf** [PrestaShop/ps_faviconnotificationbo#3]
@clementdaubeuf 
Color picker isn't correctly displayed

![capture d ecran_352](https://user-images.githubusercontent.com/13449658/46207397-88bc9000-c327-11e8-8507-fbdabddba6a7.png)

**Comment by khouloudbelguith** [comment]
Hi @marionf,

Here's the PR: https://github.com/PrestaShop/PrestaShop/pull/10753 to fix the issue.

Best regards, Khouloud

**Comment by clementdaubeuf** [comment]
I close the issue as it's fixed.

Thanks @marionf & @khouloudbelguith

**Comment by eternoendless** [comment]
The fix should be performed in the module, not in the core. Khouloud's PR for more info.

**Comment by Shiryu75** [comment]
Why ? The problem in jquery colorpicker prevents the detection of the PS_BASE_URI and is affecting all shops.

**Comment by PierreRambaud** [comment]
@Shiryu75 Because the jquery colorpicker plugin allows you set the default img location explicitely, as the core does: https://github.com/PrestaShop/PrestaShop/blob/develop/admin-dev/themes/default/template/helpers/form/form.tpl#L904-L906

**Comment by Shiryu75** [comment]
This is only working if you use HelperForms which we don't

**Comment by PierreRambaud** [comment]
No, if you're using jquery colorpicker you can add everywhere you want and it will work =)
```js
$.fn.mColorPicker.defaults.imageFolder = PATH_TO_YOUR_IMAGES_DIRECTORY;
```

**Comment by marionf** [comment]
Fixed by #6
